app/models/post.rb

```diff
@@ -13,6 +13,6 @@ def update_post_counter
   end
 
   def recent_comments
-    comments.order(created_at: :asc).limit(5)
+    comments.order(created_at: :asc).includes(:post).limit(5)
   end
```

Comment:
- [ ] Please make sure to solve the "N+1" problem when fetching all posts and their comments for a user in the controllers not in models.
- Example in the posts#index action you can use 
`@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`